runtime/j9vm/java11vmi.c

```diff
@@ -851,6 +933,29 @@ JVM_DefineModule(JNIEnv * env, jobject module, jboolean isOpen, jstring version,
 #endif /* defined(CALL_BUNDLED_FUNCTIONS_DIRECTLY) */
 	vmFuncs->internalExitVMToJNI(currentThread);
 
+#if JAVA_SPEC_VERSION >= 15
+cleanup:
+	if (NULL != packages) {
+		jsize pkgIndex = 0;
+		for (pkgIndex = 0; pkgIndex < numPackages; pkgIndex++) {
+			jobject packageObj = NULL;
+			const char* packageName = packages[pkgIndex];
+			if (NULL == packageName) {
+				break;
+			}
+
+			packageObj = (*env)->GetObjectArrayElement(env, packageArray, pkgIndex);
+			if ((*env)->ExceptionCheck(env)) {
+				Assert_SC_unreachable();
+				break;
+			}
+			(*env)->ReleaseStringUTFChars(env, packageObj, packageName);
+			(*env)->DeleteLocalRef(env, packageObj);
```

Comment:
same here - keep vm access and use the vm helpers rather than JNI

Reply:
Updated to use copyStringToUTF8Helper().